scss/button/_layout.scss

```diff
@@ -232,6 +233,7 @@
         &::before {
             content: "";
             background-color: currentColor;
+            @include border-radius();
```

Comment:
Thus far, only theme default seems like an exception. We could override `border-radius` only in the theme file.

If at a later stage we deem necessary we can introduce a variable.